CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.2.2)
 
 project(QOwnNotes)
-
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Why was this added?

Reply:
Oops I forgot I added that lol, I'll remove it

Reply:
Removed it now